### ISSUE TITLE
Whether you plan to write a document?

### ISSUE THREAD

**Issue by huang12zheng** [lesnitsky/microfrontends#1]
Whether you plan to write a document?
Let the reader know what it's for. Or a road map

**Comment by lesnitsky** [comment]
Yes, I plan to write a doc. However, I'm not entirely sure if the API would stay as it is right now or whether it will have a dramatic change. Everything you need to know right now is on readme – it is an experimental project, don't use it in production 😉 (or use at your own risk).

The roadmap is unclear. The code will be covered with docs and announced to the public when I will have a feeling I'm satisfied with using it in my own apps. Currently, I still have the "this is a bit unfinished" vibe and keep adding more helpers to boost my productivity. 

In case you want to use it – [examples folder](https://github.com/lesnitsky/microfrontends) should give you an idea of how to use it. Let me know if you're interested in any other details. You're also welcome to help me with the documentation
